torch/ao/quantization/fx/lstm_utils.py

```diff
@@ -104,7 +104,8 @@ def make_qconfig(obs_ctr: _PartialWrapper) -> QConfig:
     # Insert observers into each LSTM cell
     # TODO: maybe make this work for layer_bw as well
     for layer in quantizable_lstm.layers:
-        cell = layer.layer_fw.cell
+        cell = layer.layer_fw.cell  # type: ignore[union-attr]
+        assert isinstance(cell, torch.nn.Module)
```

Comment:
Please add nice error message here as well saying that cell should be a Module, did you monkey patch it?